tests/regression/test_regression.py

```diff
@@ -87,11 +87,11 @@ def _test_regression(
 class TestMultiClassCls(BaseTest):
     # Test case parametrization for model
     MODEL_TEST_CASES = [  # noqa: RUF012
-        ModelTestCase(task="classification", name="otx_deit_tiny"),
-        ModelTestCase(task="classification", name="otx_dino_v2"),
-        ModelTestCase(task="classification", name="otx_efficientnet_b0"),
-        ModelTestCase(task="classification", name="otx_efficientnet_v2"),
-        ModelTestCase(task="classification", name="otx_mobilenet_v3_large"),
+        ModelTestCase(task="multi_class_classification", name="otx_deit_tiny"),
```

Comment:
![image](https://github.com/openvinotoolkit/training_extensions/assets/26541465/af0b8551-0089-420d-b059-a9bc45aeedc6)
You've not changed this directory in this PR.

Reply:
Ah, thanks :)

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2739/commits/b6beac6aeb95277e4a8a95eb2026ea78f43f8687